pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -431,7 +431,7 @@ func aclToRules(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) (types.IPTabl
 	}
 	// The same rules as above for IPv4.
 	// If we have a bridge service then bridgeIP might be "".
-	if aclArgs.IPVer == 4 && aclArgs.BridgeIP != "" {
+	if aclArgs.IPVer == 4 && aclArgs.NIType != types.NetworkInstanceTypeSwitch {
```

Comment:
Does the function ever get called before BridgeIP is set for the other NI types? Or does all the callers check that BridgeIP is set?

Reply:
This function is called during app instance creation and bridge IP allocation (for other then switch instance) happens before during network instance creation.
I'll fix these checks anyway to be more explicit.